test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -1,36 +1,14 @@
-import signal
+import time
+from typing import Any, Dict
 
+import pytest
 import stomp
-from brewtils.models import Operation, Request
+from brewtils.models import Operation
+from brewtils.rest.easy_client import EasyClient
 from brewtils.schema_parser import SchemaParser
-import pytest
-import time
-import json
-
-from brewtils.errors import ValidationError
-
-try:
-    from helper import delete_plugins
```

Comment:
None of these were in use. But even if they had been used, the syntax appears to be wrong. `helper` cannot be imported directly, as far as I can tell. The relative import statement is the correct one (if `helper` code was needed in this file).